// @alicloud/tea-console/dist/client.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
class Client {
    /**
     * Console val with log level
     * @param val the printing string
     * @return void
     * @example [LOG] tea console example
     */
    static log(val) {
        console.log('[LOG] ' + val);
    }
    /**
     * Console val with info level
     * @param val the printing string
     * @return void
     * @example [INFO] tea console example
     */
    static info(val) {
        console.log('[INFO] ' + val);
    }
    /**
     * Console val with warning level
     * @param val the printing string
     * @return void
     * @example [WARNING] tea console example
     */
    static warning(val) {
        console.log('[WARNING] ' + val);
    }
    /**
     * Console val with debug level
     * @param val the printing string
     * @return void
     * @example [DEBUG] tea console example
     */
    static debug(val) {
        console.log('[DEBUG] ' + val);
    }
    /**
     * Console val with error level
     * @param val the printing string
     * @return void
     * @example [ERROR] tea console example
     */
    static error(val) {
        console.log('[ERROR] ' + val);
    }
}
exports.default = Client;
